src/0_storage.js

```diff
@@ -17,7 +19,8 @@ var storage;
 
 if (TITANIUM_BUILD) {
 	/** @typedef {{listProperties: function(), setString: function({string}, {string}), getString: function({string})}}*/
-	Ti.App.Properties;
+	// todo(dmitri): what does this line do? do you actuall mean `prop = Ti.App.Properties;`?
```

Comment:
Yeah you can always use https://github.com/BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK/issues for that, and tag the person and have a reference to the code you're talking about. (@Kirkkt if you click on the original line number, it will give you an anchored url to that line in that file i.e. https://github.com/BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK/pull/99/files#diff-6fafd7c1be3ed1a7821a1e31716d7fd0L20)